runtime/lib/backend/lightning/lightning_kokkos/LightningKokkosSimulator.cpp

```diff
@@ -32,7 +32,7 @@ auto LightningKokkosSimulator::AllocateQubits(size_t num_qubits) -> std::vector<
     const size_t cur_num_qubits = this->device_sv->getNumQubits();
     const size_t new_num_qubits = cur_num_qubits + num_qubits;
     this->device_sv = std::make_unique<StateVectorT>(new_num_qubits);
```

Comment:
Shouldn't this line also be an issue? This line is just creating a new state vector without copying over the previous data?